### ISSUE TITLE
Staging Review finding: User is not informed of confirmation value

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#123444]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Horizon
**Product:** MHV on VA.gov
**Feature:** Notification email update prompts

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the profile screen, the confirmation alert does not indicate the email address that is being confirmed. On the /my-health screen, the email is provided in the alert, so the user understands what they are confirming. On the profile screen, the user is being asked to confirm information that isn't present in the alert. For users of assistive technology, zoomed in users, older users, and users with cognitive disabilities, the association between the alert and the email field below it may not be immediately clear.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1492" height="1098" alt="contact email below" src="https://github.com/user-attachments/assets/bbe4da6a-d56f-49ae-825c-748fdc7088ad" />

</details>

## Recommended action

Include what is being confirmed (the email address) in the confirmation alert. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
